### PR TITLE
Add log info

### PR DESCRIPTION
**Pull request by ellisrichardj**
I have added some minimal log info (user, commit id) to the output file.  I have used the inbuilt nextflow properties and appended them to the combined csv.  I think this will be useful for traceability and accreditation.

### CODE REVIEW

File: bin/combineCsv.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import argparse
 from datetime import datetime, date
 
-def combine(assigned_csv, bovis_csv, seq_run, read_threshold, abundance_threshold):
+def combine(assigned_csv, bovis_csv, seq_run, user, commit, read_threshold, abundance_threshold):
```
Comment (afishman) [inline bin/combineCsv.py:10]: We can get this information directly from python (requires gitpython):
```
import getpass
import git
user = getpass.getuser()

repo = git.Repo(search_parent_directories=True)
sha = repo.head.object.hexsha
```

See: https://stackoverflow.com/questions/14989858/get-the-current-git-hash-in-a-python-script
and: https://stackoverflow.com/questions/842059/is-there-a-portable-way-to-get-the-current-username-in-python

File: bin/combineCsv.py [hunk 2]
```diff
@@ -38,11 +38,18 @@ def combine(assigned_csv, bovis_csv, seq_run, read_threshold, abundance_threshol
     #Write to csv
     finalout_df.to_csv("{}_FinalOut_{}.csv".format(seq_run, date_out))
 
+    #Append log info
+    with open("{}_FinalOut_{}.csv".format(seq_run, date_out), "a") as outFile:
```
Comment (afishman) [inline bin/combineCsv.py:42]: Appending non-csv data to a csv file will make the file cumbersome to parse. 

How about saving this to a new file instead (e.g. `metadata.json` or `metadata.txt`)?

File: bin/readStats.bash [hunk 3]
```diff
@@ -50,7 +50,7 @@ pair_id=$1
     
     if [[ ${pc_aft_trim%%.*} -lt $minafttrim ]]; then flag="LowQualData"
         elif [[ ${avg_depth%%.*} -ge $mindepth ]] && [[ ${pc_mapped%%.*} -gt $minpc ]]; then flag="Pass"
-        elif [[ ${avg_depth%%.*} -lt $mindepth ]] && [[ ${pc_mapped%%.*} -lt $minpc ]] && [[ $num_trim -gt $minreads ]]; then flag="Comtaminated"
+        elif [[ ${avg_depth%%.*} -lt $mindepth ]] && [[ ${pc_mapped%%.*} -lt $minpc ]] && [[ $num_trim -gt $minreads ]]; then flag="Contaminated"
```
Comment (afishman) [inline bin/readStats.bash:53]: Great to see this spelling corrected! Need to update the unit test to make them pass. `tests/jobs/lod.bash` line 16

**Review by afishman — CHANGES_REQUESTED** [review]

File: bTB-WGS_process.nf [hunk 4]
```diff
@@ -337,12 +337,12 @@ process CombineOutput {
 	file '*.csv' into FinalOut
 
 	"""
-	combineCsv.py Assigned.csv Qbovis.csv ${params.DataDir}
+	combineCsv.py Assigned.csv Qbovis.csv ${params.DataDir} ${workflow.userName} ${workflow.commitId}
```
Comment (afishman) [inline bTB-WGS_process.nf:340]: Simpler to have less input args - can get this info from python (see below)

**Review by afishman — CHANGES_REQUESTED** [review]

File: bin/combineCsv.py (hunk 1 above)
Comment (ellisrichardj) [reply bin/combineCsv.py:10]: Thanks for this.  I have added an extra line to ensure that the repo sha is captured when running the script from a different directory

File: bin/combineCsv.py (hunk 2 above)
Comment (ellisrichardj) [reply bin/combineCsv.py:42]: As discussed have kept like this to ensure tracability for accrediation

File: bin/readStats.bash (hunk 3 above)
Comment (ellisrichardj) [reply bin/readStats.bash:53]: Good spot - now fixed

File: bTB-WGS_process.nf (hunk 4 above)
Comment (ellisrichardj) [reply bTB-WGS_process.nf]: Have changed to your suggestion

Comment (ellisrichardj) [reply bTB-WGS_process.nf]: Changed as suggester

**Review by ellisrichardj — COMMENTED** [review]
Made changes as agreed

**Review by afishman — APPROVED** [review]
Fantastic work. Just remember to use underscores when naming python variables in the future. Strictly speaking `scriptpath` should probably be `script_path`

**Pull request merged**